docs/zh_cn/build/windows.md

```diff
@@ -0,0 +1,336 @@
+  </tr>
+  <tr>
+    <td>MMDEPLOY_TARGET_BACKENDS</td>
+    <td>{"trt", "ort", "pplnn", "ncnn", "openvino"}</td>
+    <td>N/A</td>
+    <td> <b>默认情况下，SDK不设置任何后端</b>, 因为它与应用场景高度相关。 当选择多个后端时， 中间使用分号隔开。比如，<pre><code>-DMMDEPLOY_TARGET_BACKENDS="trt;ort;pplnn;ncnn;openvino"</code></pre>
+    构建时，几乎每个后端，都需设置一些环境变量，用来查找依赖包。<br>
+    1. <b>trt</b>: 表示 TensorRT, 需要设置 TENSORRT_DIR 和 CUDNN_DIR。类似， <pre><code>-DTENSORRT_DIR={tensorrt_dir}<br>-DCUDNN_DIR={cudnn_dir}</code></pre>
+    2. <b>ort</b>: 表示 ONNXRuntime，需要设置 ONNXRUNTIME_DIR。类似， <pre><code>-DONNXRUNTIME_DIR={onnxruntime_dir}</code></pre>
+    3. <b>pplnn</b>: 表示 PPL.NN，需要设置 pplnn_DIR。<b>当前版本尚未验证</b> <br>
+    4. <b>ncnn</b>：需要设置 ncnn_DIR。<b>当前版本尚未验证</b> <br>
+    5. <b>openvino</b>: 表示 OpenVINO，需要设置 InferenceEngine_DIR。<b>当前版本尚未验证通过</b>
+   </td>
+  </tr>
+  <tr>
+    <td>MMDEPLOY_CODEBASES</td>
+    <td>{"mmcls", "mmdet", "mmseg", "mmedit", "mmocr", "all"}</td>
+    <td>N/A</td>
+    <td>用来设置SDK后处理组件，加载OpenMMLab算法仓库的后处理功能。已支持的算法仓库有'mmcls'，'mmdet'，'mmedit'，'mmseg'和'mmocr'。如果选择多个codebase，中间使用分号隔开。比如，<code>-DMMDEPLOY_CODEBASES="mmcls;mmdet"</code>。也可以通过 <code>-DMMDEPLOY_CODEBASES=all</code> 方式，加载所有codebase。</td>
+  </tr>
+  <tr>
+    <td>BUILD_SHARED_LIBS</td>
+    <td>{ON, OFF}</td>
+    <td>ON</td>
+    <td>动态库的编译开关。设置OFF时，编译静态库</td>
+  </tr>
+</tbody>
+</table>
+
+
+##### 编译样例
+
+下文展示2个构建SDK的样例，分别用于不同的运行环境。
+
+- cpu + ONNXRuntime
+
+  ```PowerShell
+  mkdir build
+  cd build
+  cmake .. -G "Visual Studio 16 2019" -A x64 -T v142 `
+      -DMMDEPLOY_BUILD_SDK=ON `
+      -DMMDEPLOY_TARGET_DEVICES="cpu" `
+      -DMMDEPLOY_TARGET_BACKENDS="ort" `
+      -DMMDEPLOY_CODEBASES="all" `
+      -DONNXRUNTIME_DIR={onnxruntime_dir} `
+      -Dspdlog_DIR={spdlog_dir}/build/install/lib/cmake/spdlog `
+      -DOpenCV_DIR={opencv_dir}/build
+  cmake --build . --config Release -- /maxcpucount:4
+  cmake --install . --config Release
+  ```
+
+- cuda + TensorRT
+
+  ```PowerShell
+   mkdir build
+   cd build
+   cmake .. -G "Visual Studio 16 2019" -A x64 -T v142 `
+     -DMMDEPLOY_BUILD_SDK=ON `
+     -DMMDEPLOY_TARGET_DEVICES="cuda" `
+     -DMMDEPLOY_TARGET_BACKENDS="trt" `
+     -DMMDEPLOY_CODEBASES="all" `
+     -Dpplcv_DIR={pplcv_dir}/pplcv-build/install/lib/cmake/ppl `
+     -DTENSORRT_DIR={tensorrt_dir} `
+     -DCUDNN_DIR={cudnn_dir} `
+     -Dspdlog_DIR={spdlog_dir}/build/install/lib/cmake/spdlog `
+     -DOpenCV_DIR={opencv_dir}/build
+   cmake --build . --config Release -- /maxcpucount:4
+   cmake --install . --config Release
+  ```
+- 其他
+
+  请参考上述两个示例，以及前述 SDK 的编译选项，在其他运行环境下编译 SDK
+
+##### 编译 SDK Demo
+
+```PowerShell
+cd install/example
+mkdir build
+cd build
+cmake .. -G "Visual Studio 16 2019" -A x64 -T v142 `
+  -DMMDeploy_DIR={mmdeploy_dir}/build/install/lib/cmake/MMDeploy `
+  -Dspdlog_DIR={spdlog_dir}/build/install/lib/cmake/spdlog `
+  -DOpenCV_DIR={opencv_dir}/build
+cmake --build . --config Release -- /maxcpucount:4
+
+$env:path = "${mmdeploy_dir}/build/install/bin;" + $env:path
+
+```
```

Comment:
Maybe we can give an example about how to run the dmeo in the future?

Reply:
It will be shown in `get_started.md`